### PR TITLE
Applist Page is working as expected

### LINKED ISSUE #13: Clear bugs in Susi Apps Page
Actual Behaviour
There are some js bugs, you can look at [here](https://jyoth1raditya.github.io/accounts.susi.ai/apps/applist/index.html)
image
Expected Bahaviour
The actual behaviour should be like [this](http://api.susi.ai/apps/applist/index.html)
image

### PR DESCRIPTION
**Pull request by jyoth1raditya**
Fixes #13, Applist page is working. Here is the image of page.
image

You can test the page : https://jyoth1raditya.github.io/accounts.susi.ai/apps/applist/index.html
(Please hit `load unsafe scripts` on the top. This problem will end once we host `accounts.susi.ai` on our custom domain)

### CODE REVIEW

File: js/framework.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 var app = angular.module("loklak", ['ngRoute']);
 app.controller("status", function($scope, $http) {
-  $http.get("/aggregation/status.json").
+  $http.get("http://api.susi.ai/aggregation/status.json").
```
Comment (DravitLochan) [inline js/framework.js]: This end point seems to be broken. @Orbiter Please see

File: js/framework.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ app.controller("search", function($scope, $http) {
   $scope.search = function() {
     if ($scope.query != '') {
       $scope.results = [];
-      $http.get("/aaa/search.json?q=" + $scope.query).
+      $http.get("http://api.susi.ai/aaa/search.json?q=" + $scope.query).
```
Comment (DravitLochan) [inline js/framework.js]: This end point also gives error page.

**Comment by DravitLochan** [comment]
@mariobehling Since the above mentioned end points are causing any problem while rendering the page, please merge this!

**Comment by jyoth1raditya** [comment]
@mariobehling Please merge this as we need this commit to progress further

**Comment by mariobehling** [comment]
@jyoth1raditya @DravitLochan As we now run this project in this PR please also follow the same process as elsewhere. I see that this PR has not been approved.

Before a PR can get merged please ensure you have two contributors approving a PR. Please use the "@" feature to ask for reviews and collaborate on other channels, e.g. chat to get the reviews faster and speed up the process.

**Comment by jyoth1raditya** [comment]
@dynamitechetan please review

**Comment by jyoth1raditya** [comment]
@saurabhjn76 please review

**Comment by jyoth1raditya** [comment]
@chashmeetsingh @isuruAb  Please review

**Pull request merged**